compiler/src/test/scala/play/twirl/compiler/test/CompilerSpec.scala

```diff
@@ -87,12 +87,36 @@ abstract class CompilerTests(oldParser: Boolean = false) extends Specification {
       set must be_==("firstsecondthird")
     }
 
-    "compile successfully (imports)" in {
```

Comment:
And here's the template that's being tested: https://github.com/richdougherty/twirl/blob/import-shadowing/compiler/src/test/resources/argImports.scala.html